### PR TITLE
Fix difficult translation for other languages

### PR DESCRIPTION
**Pull request by lunny**
Fix https://github.com/go-gitea/gitea/pull/24051#discussion_r1163697643

### CODE REVIEW

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -1710,7 +1710,7 @@ pulls.delete.text = Do you really want to delete this pull request? (This will p
 
 milestones.new = New Milestone
 milestones.closed = Closed %s
-milestones.updated = Updated
+milestones.updated = Updated %s
```
Comment (wxiaoguang) [inline options/locale/locale_en-US.ini]: The key could be `updated_on`

File: templates/repo/issue/milestones.tmpl [hunk 2]
```diff
@@ -88,7 +88,7 @@
 							{{svg "octicon-check" 16 "gt-mr-3"}}
 							{{LocaleNumber .NumClosedIssues}}&nbsp;{{$.locale.Tr "repo.issues.closed_title"}}
 							{{if .TotalTrackedTime}}{{svg "octicon-clock"}} {{.TotalTrackedTime|Sec2Time}}{{end}}
-							{{if .UpdatedUnix}}{{svg "octicon-clock"}} {{$.locale.Tr "repo.milestones.updated"}} {{TimeSinceUnix .UpdatedUnix $.locale}}{{end}}
+							{{if .UpdatedUnix}}{{svg "octicon-clock"}} {{$.locale.Tr "repo.milestones.updated" (TimeSinceUnix .UpdatedUnix $.locale)}}{{end}}
```
Comment (wxiaoguang) [inline templates/repo/issue/milestones.tmpl]: I do not think it works .... do you have a screenshot?

File: options/locale/locale_en-US.ini (hunk 1 above)
Comment (yardenshoham) [reply options/locale/locale_en-US.ini]: Originally it was `milestones.update_ago`

Comment (lunny) [reply options/locale/locale_en-US.ini]: To keep consistent with `closed`, so `updated` is also accepted?

Comment (wxiaoguang) [reply options/locale/locale_en-US.ini]: A clear key could help translators a lot.

If the key is `updated_on`, the translators could know it's for date/time

But if the key is `updated`, the translators have to guess: update by someone? updated to something? updated on someday?

I really prefer to make every name exactly clear.

File: templates/repo/issue/milestones.tmpl [hunk 3]
```diff
@@ -88,7 +88,7 @@
 							{{svg "octicon-check" 16 "gt-mr-3"}}
 							{{LocaleNumber .NumClosedIssues}}&nbsp;{{$.locale.Tr "repo.issues.closed_title"}}
 							{{if .TotalTrackedTime}}{{svg "octicon-clock"}} {{.TotalTrackedTime|Sec2Time}}{{end}}
-							{{if .UpdatedUnix}}{{svg "octicon-clock"}} {{$.locale.Tr "repo.milestones.updated"}} {{TimeSinceUnix .UpdatedUnix $.locale}}{{end}}
+							{{if .UpdatedUnix}}{{svg "octicon-clock"}} {{$.locale.Tr "repo.milestones.updated" (TimeSinceUnix .UpdatedUnix $.locale)  | Safe}}{{end}}
```
Comment (yardenshoham) [inline templates/repo/issue/milestones.tmpl]:
```suggestion
							{{if .UpdatedUnix}}{{svg "octicon-clock"}} {{$.locale.Tr "repo.milestones.updated" (TimeSinceUnix .UpdatedUnix $.locale) | Safe}}{{end}}
```

File: options/locale/locale_en-US.ini (hunk 1 above)
Comment (wxiaoguang) [reply options/locale/locale_en-US.ini]: At least, please revert to the old `update_ago` if the text doesn't change, then translators do not need to do unnecessary re-translating.

File: templates/repo/issue/milestones.tmpl (hunk 2 above)
Comment (yardenshoham) [reply templates/repo/issue/milestones.tmpl]: Fixed in https://github.com/go-gitea/gitea/pull/24070/commits/729f1742fc480462c4d523a130eb4535c9eaca62

**Review by delvh — APPROVED** [review]

**Review by yardenshoham — APPROVED** [review]

**Review by silverwind — APPROVED** [review]

**Comment by silverwind** [comment]
.

**Pull request merged**